aws-env.dev

```diff
@@ -0,0 +1,8 @@
+##this file store all evronment variables.
```

Comment:
Replace with
`## This file store all environment variables for the DEV environment